index-africa

```diff
@@ -0,0 +1,31 @@
+---
+layout: subsite-galaxy
+website: https://africa.usegalaxy.eu
+subdomain: africa
+---
+
+# Welcome to africa.usegalaxy.eu, the African galaxy server
+
+This subdomain of africa.usegalaxy.eu is devoted to assist researchers on the African continent accelerate their genomics research and analyses.
+
+<br/>
+<img src="https://avatars0.githubusercontent.com/u/68950141?s=200&v=4" height="100px" alt="ASaiM logo"/>
+
+Welcome to **Galaxy Africa** Docs in the making.
+
+
+1. TOC
+{:toc}
+
+
+# Get started
+
+Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour](https://galaxyproject.github.io/training-material/){:target="_blank"} through Galaxy's user interface.
+
+
+# Tools
+# Tutorials
+# Workflows
+# References
+
+{% bibliography --cited --prefix index-africa --group_by none %}
```

Comment:
This means you can add here https://github.com/usegalaxy-eu/website/tree/master/_bibliography your own BibTeX file and it will be rendered at the website for you.